docs/user/tutorial/40-people.rst

```diff
@@ -49,7 +49,11 @@ To create a new Person:
         Description
             Freda oversees all technical activities in our workshop.
 
-    .. todo:: screenshot of above
+.. image:: /user/tutorial/images/create_new_person.png
```

Comment:
sure it is, but that would mean that we have to cut the images a bit more. Which can have the effect that the user doesn't understand where that step happens because of the lack of information on the screenshot. And please consider that scrolling is a natural concept of getting information while clicking to enlarge an image that is too small can also result in bad ergonomics.